tests/unit/nupic/encoders/adaptivescalar_test.py

```diff
@@ -29,104 +29,117 @@
 from nupic.encoders.adaptivescalar import AdaptiveScalarEncoder
 from nupic.encoders.adaptivescalar_capnp import AdaptiveScalarEncoderProto
 
-############################################################################
+
+
 class AdaptiveScalarTest(unittest.TestCase):
   """Tests for AdaptiveScalarEncoder"""
 
 
   def setUp(self):
-    # forced: it's strongly recommended to use w>=21, in the example we force skip the check for readibility
-    self._l = AdaptiveScalarEncoder(name='scalar', n=14, w=5, minval=1, maxval=10,
-                              periodic=False, forced=True)
+    # forced: it's strongly recommended to use w>=21, in the example we force
+    # skip the check for readibility
+    self._l = AdaptiveScalarEncoder(name="scalar", n=14, w=5, minval=1,
+                                    maxval=10, periodic=False, forced=True)
 
   def testMissingValues(self):
     """missing values"""
-    # forced: it's strongly recommended to use w>=21, in the example we force skip the check for readib.
-    mv = AdaptiveScalarEncoder(name='mv', n=14, w=3, minval=1, maxval=8, periodic=False, forced=True)
+    # forced: it's strongly recommended to use w>=21, in the example we force
+    # skip the check for readib.
+    mv = AdaptiveScalarEncoder(name="mv", n=14, w=3, minval=1, maxval=8,
+                               periodic=False, forced=True)
     empty = mv.encode(SENTINEL_VALUE_FOR_MISSING_DATA)
-    print "\nEncoded missing data \'None\' as %s" % empty
     self.assertEqual(empty.sum(), 0)
 
+
   def testNonPeriodicEncoderMinMaxSpec(self):
     """Non-periodic encoder, min and max specified"""
 
-    self.assertTrue((self._l.encode(1) == numpy.array([1, 1, 1, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0],
-                                       dtype=defaultDtype)).all())
-    self.assertTrue((self._l.encode(2) == numpy.array([0, 1, 1, 1, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0],
-                                       dtype=defaultDtype)).all())
-    self.assertTrue((self._l.encode(10) == numpy.array([0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 1, 1, 1, 1],
-                                        dtype=defaultDtype)).all())
+    self.assertTrue(numpy.array_equal(
+      self._l.encode(1),
```

Comment:
`==` is element-wise, so you get another list back, which means you have to call `all()` to check if they are all equal element-wise.  `numpy.array_equal(A, B)` is more readable than `(A == B).all()`